documentai/snippets/handle_response_sample_v1beta3.py

```diff
@@ -30,8 +30,52 @@
 # mime_type = "application/pdf" # Refer to https://cloud.google.com/document-ai/docs/file-types for supported file types
 
 # [END documentai_process_custom_extractor_document]
+# [END documentai_process_summarizer_document]
+
+
+# [START documentai_process_layout_document]
+def process_document_layout_sample(
+    project_id: str,
+    location: str,
+    processor_id: str,
+    processor_version: str,
+    file_path: str,
+    mime_type: str,
+) -> documentai.Document:
+    process_options = documentai.ProcessOptions(
+        layout_config=documentai.ProcessOptions.LayoutConfig(
+            chunking_config=documentai.ProcessOptions.LayoutConfig.ChunkingConfig(
+                chunk_size=1000,
+                include_ancestor_headings=True,
+                breakpoint_percentile_threshold=90,
+            )
+        )
+    )
+
+    # Online processing request to Document AI
```

Comment:
Comments shouldn't be used to re-iterate what the code does, but to add additional explanation.

Reply:
Removed